### DISCUSSION TITLE
How I can configure default assistants for users?

### DISCUSSION THREAD

**Discussion by unameisfine** [AllYourBot/hostedgpt#514] [General]
Can I do it now? For me, it is useful for automatically sharing the latest models (like o1) with my friends — users of my instance. They are not computer geeks to add all these models manually. This application is a great way to share LLMs with parents and friends.

**Comment by keithschacht** [comment]
I really appreciate your interest in this feature and in this project. You and me and a handful of other people are using it regularly. But I’m finding myself in an interesting place with this project, trying to decide what to do next with it… I had three motivations for building this. The first was that I wanted it for myself. I spec’d out all the features that were most important to me and I implemented them. The second was that I wanted go deep on a few technologies to thoroughly understand them myself. (new LLMs, new Rails features, etc). And the third was that I wanted to create a product which I thought could have a potentially large audience of other people excitedly using it and contributing to it.

I’ve definitely accomplished 1 and 2 but my biggest miss has been 3. I haven’t created something that lots of people are excited to use. I was excited about doing an initial push with this project to build momentum, and I’ve done that. But my hope was that this initial push would lead to thousands or at least hundreds of people using the app regularly. My hope was that I’d dozens of contributors on any given week. But the actual numbers and impact of this project is much smaller than I was hoping for.  There are probably about 5 other people who use it regularly, maybe as many as 10. But that’s about it. There are also a handful of people who have helped contribute to this project, but on any given week there is maybe 1 other person besides me writing some code.

I find myself continually not prioritizing adding the small features and fixing the small bugs that exist in this project unless they’re at the top of my personal list. With regards to this specific feature you’re asking me about, I feel like, “I could prioritize this next, but … I want to prioritize doing the work that will lead to creating something that a lot more people are excited to use!” I don’t think this and a handful of other features are going to be what makes the difference. BUT, I haven’t figured out what will.

I am considering some larger pivots of this project (or realistically I’d do forks, so I wouldn’t actually change this specific HostedGPT). I find myself thinking about starting a new project which is related to HostedGPT but is different. Other things that I want for myself, which I think may be interesting to a larger number of other people.

In short, I’m undecided on what I should prioritize next for my own goals of creating something lots of people want. That’s what I’m trying to figure out… And this is a long-winded way of saying: I’m not sure how to prioritize this specific feature you’re asking about, but I don’t feel like I have enough other people who want it. I know this isn’t the answer you were hoping for. :)

**Reply by unameisfine** [reply]
The voting system could be informative about user count and priorities.

**Reply by keithschacht** [reply]
If you know rails you’re welcome to take the lead on this and I can review PR and give you advice

**Comment by keithschacht** [comment]
FYI, new models can now be created for other users by editing the `models.yml` file. Changes will be applied when the server is restarted. I just added this to the README under Optional Features
